### PR TITLE
Milestone 4: Desktop version

### PR DESCRIPTION
**Pull request by BenMKT**
- Used media queries in my CSS file to make desktop version ` (Desktop: >=768px) `.
- To lay out the elements on the page used Grid and Flexbox.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -39,47 +50,638 @@ body {
...
+  }
+
+  #headerarticle {
+    display: flex;
+    flex-direction: column;
+    margin-left: 260px;
+    width: 920px;
+    margin-bottom: 80px;
+  }
+
```
Comment (divinecharlotte) [inline styles.css]: - [ ] As you move from the mobile version to the desktop version kindly consider sticking to the Figma design as much as possible kindly consider fixing this section on the mobile version by adding well the background background image to make your website an exact duplicate of the given design on both screens check the images below to see the difference 
- your design 👇 
Screen Shot 2023-06-08 at 20 04 03

- Given design 👇 
Screen Shot 2023-06-08 at 20 07 04

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @BenMKT    , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Use of GitHub flow👍 
- Descriptive pr 👍 

 




 

## Required Changes ♻️

- [x] Kindly consider adding a working link to your social media under the author's section to give a good user experience to the person who would like to reach you out 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (BenMKT) [reply styles.css]: - Fixed header mobile background to conform to Figma design.
- Added a working link to my GitHub and social media under the author's section to give a good user experience.

File: README.md [hunk 2]
```diff
@@ -144,8 +144,8 @@ Example:
 
 👤 **Benson Kiai:**
 
-- GitHub: [@BenMKT](https://github.com/BenMKT/Portfolio)
-- linkedin: [@BenK](www.linkedin.com/in/bensonkiai)
+- GitHub: [@BenMKT](https://github.com/BenMKT/myPortfolio)
+- Linkedin: [@BenK](www.linkedin.com/in/bensonkiai)
```
Comment (victorokoroji) [inline README.md]: - [x] Your LinkedIn path is actually following github path which makes it difficult to access your LinkedIn page.

This is what I see when I click your LinkedIn;
`https://github.com/BenMKT/myPortfolio/blob/desktop-version/www.linkedin.com/in/bensonkiai`

To fix this, I think you should attach `https` to your LinkedIn route like this:
`https://www.linkedin.com/in/bensonkiai`

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @BenMKT, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- Good UI design ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (BenMKT) [reply README.md]: - Inserted `https` to my Linkedin path to work correctly in the `desktop-version` branch.

**Review by TadesseDev — APPROVED** [review]
### Hi @BenMKT 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**